spec/units/line_editor_spec.rb

```diff
@@ -0,0 +1,702 @@
+require 'spec_helper'
```

Comment:
This file is exciting. Not sure I understand all of it on a first read. Anything in particular I should focus on?